containers/layouts/LayoutsSection.js

```diff
@@ -15,115 +16,157 @@ import {
     updateComposerMode,
     updateViewMode,
     updateViewLayout,
+    updateStickyLabels,
     updateDraftType,
     updateRightToLeft
 } from 'proton-shared/lib/api/mailSettings';
+import { VIEW_MODE } from 'proton-shared/lib/constants';
 
 import DraftTypeSelect from './DraftTypeSelect';
 import TextDirectionSelect from './TextDirectionSelect';
 import ComposerModeRadios from './ComposerModeRadios';
 import ViewLayoutRadios from './ViewLayoutRadios';
 import ViewModeRadios from './ViewModeRadios';
+import StickyLabelsToggle from './StickyLabelsToggle';
+
+const { GROUP } = VIEW_MODE;
 
 const LayoutsSection = () => {
-    const [{ ComposerMode, ViewMode, ViewLayout, DraftMIMEType, RightToLeft } = {}] = useMailSettings();
+    const [{ ComposerMode, ViewMode, ViewLayout, StickyLabels, DraftMIMEType, RightToLeft } = {}] = useMailSettings();
     const { call } = useEventManager();
 
+    const { createNotification } = useNotifications();
+    const preferenceSaved = () => createNotification({ text: c('Success').t`Preference saved` });
```

Comment:
What do you think about calling the function `notifyPreferenceSaved` to make the intention more clear?

Reply:
sure!